.devcontainer/devcontainer.json

```diff
@@ -12,9 +12,14 @@
 		"--cap-add=SYS_PTRACE",
 		"--security-opt=seccomp:unconfined",
 		"--security-opt=apparmor:unconfined",
-		"--volume=/tmp/.X11-unix:/tmp/.X11-unix"
+		"--volume=/tmp/.X11-unix:/tmp/.X11-unix",
+		"--gpus" "all",
 	],
-	"containerEnv": { "DISPLAY": "${localEnv:DISPLAY}" },
+	// default
+	//"containerEnv": { "DISPLAY": "${localEnv:DISPLAY}"}
```

Comment:
Prefer to remove the commented out default lines because it will be merged into a dedicated branch

Reply:
thx!